### ISSUE TITLE
Search requests returning empty when on live server

### ISSUE THREAD

**Issue by pacificano** [rubenarakelyan/zomato-api-php#2]
Hey, wondering if you've noticed any issues when using the code on a live server? It works fine on my localhost but when put live, it seems to be connecting to zomato, doesn't throw any errors, but the search results come back empty. Just wondering if anyone else is experiencing this? I'm pretty sure its an issue on zomatos end but thought I'd check before I go crazy.

http://i.imgur.com/ai7eJ0W.png

**Comment by rubenarakelyan** [comment]
I've used this code in production without any issues. Are you sure you have all your environment variables (such as the API key) declared in your production environment as well as locally?

**Comment by pacificano** [comment]
Yeh, sorry I should have come back and closed this. Your code is very solid. 

I worked out that for some reason, a search like https://developers.zomato.com/api/v2.1/search?q=jazushi would work locally but not on a live server. But as soon as I added an entity type like https://developers.zomato.com/api/v2.1/search?entity_id=260&entity_type=city&q=‌​jazushi the search results would come back on my server. I'm not sure why. But its something that I replicated in other code, so its definitely not on your end. 

Thanks for all your work btw. Its very good. Appreciate it.

http://stackoverflow.com/questions/38034444/php-curl-request-to-zomato-api-working-on-localhost-but-only-returning-empty-arr
